addons/account/views/report_invoice.xml

```diff
@@ -210,6 +210,11 @@
                                 <small class="text-muted lh-sm"><span t-field="o.amount_total_words"/></small>
                             </p>
                         </div>
+                        <div id="display_taxes_company_currency"
```

Comment:
I'm not convinced by the architecture of that DOM

The id here is only useful for styling that div, not for identifying that div.
In our case I would only make up a class, with a sufficiently evocative yet reusable name that you could use in the scss files afterwards.

An id is only useful if we are to identify that node specifically (usually for inheritance, and there must be only one). I let you decide whether you want that div to be precisely located. Remember that each door you open for our partners/clients, you can't close back.

Reply:
Thank you very much for the review. I have made, I think, all the changes you suggest